### PR TITLE
eis updated,cms module and ais module

### PR DESCRIPTION
**Pull request by ghost**
revised push

### CODE REVIEW

**Comment by pranjulps3** [comment]
Awaiting review

**Comment by Samy-33** [comment]
Files are alright, now there will be two folders course management and similar issues. I will merge it but it has to be resolved.

File: FusionIIIT/templates/ais/ais.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    <script type="text/javascript" src="{% static 'fusion/js/editProfile.js' %}"></script>
+    <script type="text/javascript" src="{% static 'fusion/js/tablesort.js' %}"></script>
+
+    <script>
+        $(document).ready(function() {
+            $('table').tablesort();
+        });
+    </script>
+
+    <script>
```
Comment (kmunshi97) [inline FusionIIIT/templates/ais/ais.html]: Remove this Script!

File: FusionIIIT/templates/ais/ais.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
+
+{% extends 'fusion/general/base.html' %}
```
Comment (kmunshi97) [inline FusionIIIT/templates/ais/ais.html]: Change it accordingly. 
These URL's won't work!
Have a look at the code I pushed.
ASAP!

File: FusionIIIT/templates/ais/ais.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+
+{% block title %}
+    Academic Information System
+{% endblock %}
+
+
+{% block body %}
+    {% block navBar %}
+        {% include 'fusion/general/../dashboard/navbar.html' %}
```
Comment (kmunshi97) [inline FusionIIIT/templates/ais/ais.html]: Change it accordingly. 
These URL's won't work!
Have a look at the code I pushed.
ASAP!

File: FusionIIIT/templates/ais/ais.html [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        <div class="column"></div>
+
+        {% comment %}
+            The left-rail segment starts here!
+        {% endcomment %}
+        <div class="three wide column">
+
+            {% comment %}The user image card starts here!{% endcomment %}
+            {% block usercard %}
+                {% include 'fusion/general/usercard.html' %}
```
Comment (kmunshi97) [inline FusionIIIT/templates/ais/ais.html]: Change it accordingly. 
These URL's won't work!
Have a look at the code I pushed.
ASAP!

**Review by kmunshi97 — CHANGES_REQUESTED** [review]
Check all your {% extends %}, {% include %}, and {% static %} tags. 
None of them will work!
See my code for reference!

And the tablesort.js script is redundant!

File: FusionIIIT/templates/coursemanagement/addcontent.html [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+    <script type="text/javascript" src="{% static 'fusion/js/editProfile.js' %}"></script>
+    <script type="text/javascript" src="{% static 'fusion/js/tablesort.js' %}"></script>
+
+    <script>
+        $(document).ready(function() {
+            $('table').tablesort();
+        });
+    </script>
+
+    <script>
```
Comment (kmunshi97) [inline FusionIIIT/templates/coursemanagement/addcontent.html]: Remove this script everywhere!

**Review by kmunshi97 — CHANGES_REQUESTED** [review]
I don't see any urls.py and views.py.
Make them and test run on your PC, and make sure everything runs fine as you had desired.

**Comment by Samy-33** [comment]
Please don’t push urls.py and views.py that you create. They should be created by respective teams.

**Review by Samy-33 — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by ghost** [comment]
@Samy-33 @kmunshi97  Pls discuss among yourself and confirm that i have to push urls.py and views.py or not

**Pull request merged**